src/integrationTest/java/uk/gov/hmcts/reform/sendletter/config/SftpConfig.java

```diff
@@ -22,7 +24,8 @@
         };
     }
 
-    @Bean()
+    @Bean
+    @Primary
```

Comment:
why this change?

Reply:
To firstly take this configuration and not the one defined in source code. The change is related to previous comment